mungegithub/mungers/size.go

```diff
@@ -253,24 +253,26 @@ const (
 	sizeXXL = "XXL"
 )
 
+type prSize struct {
+	UpperLines int
+	Size       string
+}
+
+// This is a totally arbitrary heuristic and is open for tweaking.
```

Comment:
Add a comment that the prSize structs must be listed in order of increasing `UpperLines`.